time-series-forecast/resource/predict.R

```diff
@@ -36,18 +36,18 @@ GetForecasts <- function(ts, df, xreg = NULL, modelList, modelParameterList,
   forecastDfList <- list()
   for(modelName in names(modelList)) {
     model <- modelList[[modelName]]
-    if (modelName == "PROPHET_MODEL") {
-      freq <- ifelse(granularity == "hour", 3600, granularity)
-      future <- make_future_dataframe(model, horizon, freq, include_history = includeHistory)
-      if (!is.null(xreg)) {
-        for (c in colnames(xreg)) {
-          if (includeHistory) {
-            future[,c] <- c(df[,c], xreg[,c])
-          } else {
-            future[,c] <- xreg[,c]
-          }
-        }
-      }
+    # if (modelName == "PROPHET_MODEL") {
```

Comment:
@trivenigandhi I am getting a code validation failure on this one. I managed to get it running by commenting out lines 51-55, as well as line 100 (to avoid the additional bracket)

Reply:
Sounds good!